docs/diagrams/StorageClassDiagram.puml

```diff
@@ -10,11 +10,20 @@ class Recipe
 class Ingredient
 }
 
+package Parser {
+class RecipeFileParser
+class TitleFileParser
+}
+
+class ParseFileException
+
 Storage -up-> Ui
 Storage -down-> Recipe
 Storage -down-> Ingredient
 Storage -down-> RecipeList
+Storage ..> RecipeFileParser
+Storage ..> TitleFileParser
+Storage -left-> ParseFileException
 
 Recipe *-right-> Ingredient
```

Comment:
```suggestion
Recipe -right-> Ingredient
```
`Ingredient` is an attribute of `Recipe`, it is ok to use solid arrow here I guess?

Reply:
I think that this is a composition, rather than normal association. This is because Ingredient class cannot exist without a Recipe class, and when a Recipe class gets deleted, all the Ingredient classes attached to the Recipe class gets deleted as well.